openapi/resource-server.yaml

```diff
@@ -1321,6 +1313,13 @@ components:
     '403':
       description: Forbidden
   parameters:
+    pagination:
+      schema:
+        $ref: '#/components/schemas/pagination'
+      in: query
+      name: pagination
+      description: Pagination parameters
+      style: form
```

Comment:
`style: form` is the fix here

Reply:
:confused: 
It sounds like `form` is the default with `in: query`
https://spec.openapis.org/oas/v3.1.0#fixed-fields-9

>Describes how the parameter value will be serialized depending on the type of the parameter value. Default values (based on value of `in`): for `query` - `form`

Does that indicate there's an issue with readme.io's OpenAPI support?

Reply:
Maybe because of the fact that the parameter was referencing a schema in `components/schemas/pagination` directly with `name: pagination` it defaulted to `style: deepObject`?